sharething_pwa/src/Home/HomeScreen.tsx

```diff
@@ -19,8 +21,9 @@ class HomeScreen extends React.Component {
 
   componentDidMount() {
     this.setState({ loading: true });
-    let items = []
-     let listener = this.props.firebase.items().onSnapshot(snapshot => {
+    type items = object;
+    let items: object[] = []
+    let listener = this.props.firebase.items().onSnapshot(snapshot => {
```

Comment:
Confusing naming here. Listener is the function that you pass to `onSnapshot`. What is returned from `onSnapshot` is unsubscription function for that listener. So `unsubscribe` is a better name for that.